src/components/Header.js

```diff
@@ -1,11 +1,12 @@
-import Link from 'next/link';
+import {StyledNavButton} from './styled/StyledNavButton';
 
 export default function Header() {
 	return (
 		<header>
 			<nav>
-				<Link href="/">Home</Link>
-				<Link href="/about">About me</Link>
+				<StyledNavButton as="a" href="/timer">
+					Cooking Timer
+				</StyledNavButton>
```

Comment:
you schould use the Next Link here, not the anchor tag. It is also not necessary to wrap this link in a button, since it is a link and it should look like a link ;)